packages/design-system-docs/src/pages/components/VerticalNav/VerticalNav.example.jsx

```diff
@@ -16,6 +16,7 @@ Link.propTypes = { children: PropTypes.node };
 ReactDOM.render(
   <VerticalNav
     selectedId="team"
+    ariaNavLabel="menu"
```

Comment:
Can we change this label to something like: "Main" or "Primary"? Most screen readers will know it's a menu because it has the `<nav>` element. The label provides a bit more context. I'm going to include this point in the docs too.